src/app/service/resource/resource.service.ts

```diff
@@ -60,7 +60,10 @@ export class ResourceService extends AbstractService {
           resource_type_id: Resource.getResourceTypeId(resource),
           onesky_project_id: resource.oneskyProjectId,
           description: resource.description,
-          manifest: resource.manifest
+          manifest: resource.manifest,
+          attr_banner: resource.banner,
+          attr_banner_about: resource.bannerAbout,
+          attr_about_overview_video_youtube: resource.aboutOverviewVideoYoutube
```

Comment:
these should be using `-` instead of `_`

Reply:
hmm, this might not actually matter in the API because I see `onesky_project_id` is `onesky-project-id` within the API currently.

We will probably need to wait on the API updates to confirm this

Reply:
Yes, that's also what I thought.
In order to use property names with '-' I have to change the payload data's attribute member to a "key - value" array. I planned to change it but when I check that the API has "onesky-project-id" at the payload but the web app has "onesky_project_id" as the member name, I thought this will also work.